spanner/integration_test.go

```diff
@@ -165,6 +169,10 @@ func parseInstanceName(inst string) (project, instance string, err error) {
 	return matches[1], matches[2], nil
 }
 
+func getSpannerHost() (string) {
```

Comment:
This needs to be the following for the format check to stop complaining:

```suggestion
func getSpannerHost() string {
```

You can check the formatting by executing `gofmt -s .` in the `cloud.google.com/spanner` directory.